SECURITY.md

```diff
@@ -1,6 +1,8 @@
 # Security Policy
 Kiali takes security seriously and encourages users to report security concerns.
 
+If you run a security scan on Kiali software and would like to report a security scan report to the Kiali team, we only ask that you first verify that your scan is correctly validating the latest release and that the results are valid. Because the Kiali team takes security reports seriously, they often take priority over current work being done, and it takes the Kiali team a long time to research and validate them. So please make sure you verify that any reports you submit are reporting an accurate reflection of the Kiali software being scanned and that the security issues being reported actually affect Kiali or one of its dependencies.
```

Comment:
If you run a security scan on Kiali software and would like to report a security scan report to the Kiali team, we only ask that you first verify that your scan is correctly validating the latest release and that the results are valid. Because the Kiali team takes security reports seriously, they often take priority over current work being done, and it takes the Kiali team a long time to research and validate them. So please make sure you verify that any reports you submit are reporting an accurate reflection of the Kiali software being scanned and that the security issues being reported actually affect Kiali or one of its dependencies.

Reply:
There is no diff between your suggestion and what the original text is.

Reply:
i committed the changes suggested over in the kiali.io pr